haechan45th/app.js

```diff
@@ -149,13 +147,79 @@ app.get('/users/:userId/posts', async (req, res) => {
       WHERE users.id = ?
       `, [userId])
 
-    return res.status(200).json({gettingUserPosts})
-  } catch(error) {
+    return res.status(200).json({ gettingUserPosts })
+  } catch (error) {
     console.log(error)
-    res.status(400).json({message: "some error"})
+    res.status(400).json({ message: "some error" })
+  }
+});
+
+app.put('/posts/:userId', async (req, res) => {
+  try {
+    const { userId } = req.params;
+    const { postId } = req.body;
+    const { title, content, imageUrl, } = req.body;
+
+    const updatePosts = await dataSource.query(
+      `UPDATE posts
+      INNER JOIN users ON posts.user_id = users.id
+      SET
+      posts.title = ?,
+      posts.content = ?,
+      posts.image_url = ?
+      WHERE users.id = ? AND posts.id = ?
+      `, [title, content, imageUrl, userId, postId]
+    )
+
+    res.status(200).json({ message: 'Post updated successfully.' });
+  } catch (error) {
+    console.log(error);
+    res.status(400).json({ message: "Error has occur in USER EDITING API" })
+  }
+});
+
+app.delete('/posts/:postsId', async (req, res) => {
+  try {
+    const { postsId } = req.params;
+
+    const deletePosts = await dataSource.query(
+      `DELETE FROM posts
+      WHERE posts.id = ${postsId}
+      `
+    )
+
+    res.status(200).json({ message: 'Post DELETED successfully.' });
+  } catch (error) {
+    console.log(error);
+    res.status(400).json({ message: "Error has occur in DELETING USER POSTS" })
   }
 });
 
+app.post('/users/likes', async (req, res) => {
+  try {
+    const { user_id, postId } = req.body;
```

Comment:
```suggestion
    const { userId, postId } = req.body;
```
자바스크립트에서는 snake_case를 쓰지 않도록 해주세요! :)